portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase

```diff
@@ -371,28 +226,131 @@ definition {
 			value1 = "Title Anonymous User Ask questions");
 	}
 
-	@description = "Validate if the user can cancel the deletions."
+	@description = "Validate if the user can create a question with an AL image."
 	@priority = "4"
-	@refactorneeded
-	test CancelDeletions {
+	@refactordone
+	test CanAddQuestionWithDepotImage {
+		JSONDepot.addDepot(
+			depotDescription = "This is the description of a depot",
+			depotName = "Test Depot Name");
+
+		DepotNavigator.openDepotAdmin();
```

Comment:
@liuxingye I forgot to remove this line before sending the pull to BChan. Could you remove it in another ticket? You can use the same LPS #

Reply:
ok. I will remove it